test/react/functional-components/simple-wth-object-assign2.js

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+// the JSX transform converts to React, so we need to add it back in
+this['React'] = React;
+
+function A(props) {
+  var copyOfProps = Object.assign({}, props.bag);
+  Object.defineProperty(copyOfProps, "y", {
+    get() {
+      return 30;
+    },
+    set(x) {
+      return x;
```

Comment:
returning from a setter doesn't do anything. What is this suppose to test?

Reply:
I’m not trying to set here. I’m just trying to get

Reply:
But this code never runs, does it? We're not setting to `y` anywhere.

Reply:
We're setting `y` on `copyOfProps`.

The result is this, so it appears to be working?

`return <div>Hello 20 30</div>;`

Reply:
@gaearon @sebmarkbage I see what I think you mean now, I had a lint rule on Atom that said I needed add a setter if I had a getter, so I added a random no-op one. Haha.